workflow-dispatch/action.yml

```
@@ -0,0 +1,25 @@
+name: Workflow Dispatch
+description: Triggers a workflow with the workflow_dispatch event.
+inputs:
+  owner:
+    description: "Name of the repository owner in which the workflow resides. Defaults to the one who owns the workflow in which this action is running."
+    required: false
+  repo:
+    description: "Name of the repository in which the workflow resides. Defaults to the one in which the workflow currently running resides."
+    required: false
+  token:
+    description: "Github token. Cannot be ${{ secrets.GITHUB_TOKEN }} provided in the worflow run."
```

```suggestion
    description: "Github token to access the target repo, if private. Cannot be GITHUB_TOKEN provided in the worflow run."
```
I think that the `token` input could be `required: false`, like if you want to use it on a public repo, I don't think that this is necessary.

What do you think?

---

From what I understand after reading the documentation, this parameter is mandatory even for public repos.
> You must authenticate using an access token with the repo scope to use this endpoint. GitHub Apps must have the actions:write permission to use this endpoint. 
[source](https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#create-a-workflow-dispatch-event)

---

Thanks for the clarification!